app/javascript/packs/personal-key-page-controller.js

```diff
@@ -114,6 +115,20 @@ function hide() {
   modal.hide();
 }
 
+function downloadForIE(event) {
+  event.preventDefault();
+
+  const filename = downloadLink.getAttribute('download');
+  const data = scrapePersonalKey();
+  const blob = new Blob([data], { type: 'text/plain' });
+
+  window.navigator.msSaveBlob(blob, filename);
+}
+
 modalTrigger.addEventListener('click', show);
 modalDismiss.addEventListener('click', hide);
 formEl.addEventListener('submit', handleSubmit);
+
+if (window.navigator.msSaveBlob) {
+  downloadLink.addEventListener('click', downloadForIE);
```

Comment:
I was not able to test this myself, but @mitchellhenke said in slack the download worked